frontend/src/main/web/lib/stores/UserMatrixStore.js

```
@@ -42,8 +46,7 @@ function loadFromServer() {
       .set("Cache-Control", "no-cache, no-store, must-revalidate")
       .set("Pragma", "no-cache")
       .set("Expires", 0)
-      .end((function (res) {
-        //console.log('response:' + res.body);
+      .end(function (err, res) {
```

- [ ] should not just ignore error - handle it before anything else:

```
if (err) {
  // do something appropriate
  return
}
```